randovania/monitoring/__init__.py

```diff
@@ -153,6 +154,8 @@ def client_init() -> None:
     global_scope = sentry_sdk.Scope.get_global_scope()
     global_scope.set_tag("frozen", randovania.is_frozen())
     global_scope.set_tag("cpu.architecture", platform.machine())
+    global_scope.set_tag("cpu.processor", platform.processor())
```

Comment:
i thought you wanted to revert this?

Reply:
It works on Windows, 🤷 about not having data for linux

Reply:
Our linux userbase is only ~2% so fair enough. Worst case can always change it in the future to pull data from lscpu or something.